src/zope/testing/renormalizing.txt

```diff
@@ -234,3 +234,59 @@ Combining a checker with something else does not work:
         ...
     TypeError: unsupported operand type(s) for +: ...
 
+Using the 2to3 exception normalization:
+
+    >>> from zope.testing.renormalizing import EXCEPTION_2TO3
+    >>> checker = OutputChecker()
+    >>> want = """\
+    ... Traceback (most recent call last):
+    ... foo.bar.FooBarError: requires at least one argument."""
+    >>> got = """\
+    ... Traceback (most recent call last):
+    ... FooBarError: requires at least one argument."""
+    >>> result = checker.check_output(want, got, EXCEPTION_2TO3)
+    >>> import sys
+    >>> if sys.version < '3':
+    ...     expected = True
+    ... else:
+    ...     expected = False
+    >>> result == expected
+    True
```

Comment:
Ugh.

Maybe just do it always, on Python 2 and Python 3?  Stripping a non-existent prefix is a no-op anyway.